src/IBM.Cloud.SDK.Core/Util/Extensions/DescriptionExtension.cs

```diff
@@ -1,37 +1,37 @@
-﻿/**
-* Copyright 2017 IBM Corp. All Rights Reserved.
-*
-* Licensed under the Apache License, Version 2.0 (the "License");
-* you may not use this file except in compliance with the License.
-* You may obtain a copy of the License at
-*
-*      http://www.apache.org/licenses/LICENSE-2.0
-*
-* Unless required by applicable law or agreed to in writing, software
-* distributed under the License is distributed on an "AS IS" BASIS,
-* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-* See the License for the specific language governing permissions and
-* limitations under the License.
-*
-*/
-
-using System;
-using System.Linq;
-using System.Reflection;
-using IBM.Cloud.SDK.Core.Util.Attributes;
-
-namespace IBM.Cloud.SDK.Core.Util.Extensions
-{
-    public static class DescriptionExtension
-    {
-        //public static string Description(this Enum value)
-        //{
-        //    DescriptionAttribute attribute = value.GetType()
-        //        .GetField(value.ToString())
-        //        .GetCustomAttributes(typeof(DescriptionAttribute), false)
-        //        .SingleOrDefault() as DescriptionAttribute;
-
-        //    return attribute == null ? value.ToString() : attribute.Description;
-        //}
-    }
+﻿/**
+* Copyright 2017 IBM Corp. All Rights Reserved.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*
+*/
+
+using System;
+using System.Linq;
+using System.Reflection;
+using IBM.Cloud.SDK.Core.Util.Attributes;
+
+namespace IBM.Cloud.SDK.Core.Util.Extensions
+{
+    public static class DescriptionExtension
+    {
+        //public static string Description(this Enum value)
```

Comment:
Do we need this commented code for anything?

Reply:
hmm thats weird. I don't think I changed this. I don't see it being used so I'll delete the whole class.